cloud-run/test/run-deploy.test.js

```diff
@@ -280,6 +283,32 @@ describe('Run Deploy', () => {
     expect(exec.exec.mock.calls[0][1]).toEqual(expect.arrayContaining(['--clear-cloudsql-instances']));
   });
 
+  test('It can deploy with vpc-connector', async () => {
+    exec.exec.mockResolvedValueOnce(0);
+    setupGcloud.mockResolvedValueOnce('test-staging-project');
+    process.env.GITHUB_SHA = '382aee2'; // Not tagged
+    const service = {
+      name: 'my-service',
+      memory: '256Mi',
+      cpu: 1,
+      platform: {
+        managed: {
+          region: 'eu-west1',
+          'allow-unauthenticated': true,
+          'cloudsql-instances': [],
+          'vpc-connector': 'test-staging-project/vpc-connector',
```

Comment:
Will we have a VPC connection in each project? If yes, then I think we should declare the name without project or with * as we do with secrets.

Reply:
But I guess it's going to be on the tribe's level (in most cases)

Reply:
Would we use the same VPC connector for staging and production? If they will differ we may want to think about a solution that don't encourage teams to use two cloud-run.yaml files.

Reply:
We can't use the same connector in both prod and staging, as those have two separate Shared VPCs unless we do some kind of VPC peering (I'd prefer to avoid this)

Reply:
Why would we need a VPC connector is staging at all?